mobify/progressive-web-scaffold
# Upgrade all webpack config files to v2

(brief description / PR title)

 **JIRA**: https://mobify.atlassian.net/browse/WEB-997

## Changes
- Upgraded webpack and related dependencies.
- Configured Circle to use node 6.9.2.

## How to test-drive this PR
- Run `npm run dev`.
  - Validate that Merlin's Potions works as expected.
- Run `npm run prod:build`
  - Validate that there's no errors in the console.
